docs/community/workshops.md

```diff
@@ -0,0 +1,39 @@
+# Workshops
+Find us at the following upcoming and past workshops!
+
+## DataJoint Office Hours
+
+The DataJoint open source team will be offering monthly Office Hours.  These are public workshops on Zoom where we will offer two half-hour sessions of free hands-on support to teams using DataJoint software.
```

Comment:
I suggest removing this section in favor of a link to the support doc, to reduce maintenance. If not, suggesting a change of tense

```suggestion
The DataJoint open source team will be offering monthly Office Hours.  These are public workshops on Zoom where we offer two half-hour sessions of free hands-on support to teams using DataJoint software.
```

Reply:
Thanks @CBroz1.  I have removed this section.